ts/deploy.ts

```diff
@@ -138,7 +138,7 @@ export async function deployAll(
     );
     await waitAndRegister(rollup, "rollup", verbose);
 
-    await vault.setRollupAddress(rollup.address);
+    await waitUntilMined(vault.setRollupAddress(rollup.address));
```

Comment:
Right, although the order of `vault.setRollupAddress(rollup.address)` and `depositManager.setRollupAddress(rollup.address)` doesn't matter, we should wait until one is mined before we deploy the next so that if one deployment fails we have time to stop and think about it.